conf/deployment/ibmcloud/ibm_cloud_vpc_cluster.yaml

```diff
@@ -14,12 +14,12 @@ ENV_DATA:
   # vpc_id: VPC ID PLACEHOLDER
   # subnet_id: SUBNET ID PLACEHOLDER
   # cos_instance: COS INSTANCE PLACEHOLDER
-#AUTH:
-#  ibmcloud:
-#    api_key: IBM CLOUD API KEY PLACEHOLDER
-#    account_id: ACCOUNT ID PLACEHOLDER
-#    ibm_cos_access_key_id: KEY PLACEHOLDER
-#    ibm_cos_secret_access_key: SECRET PLACEHOLDER
+AUTH:
```

Comment:
This should not be uncommented here. You need to provide custom created config file with those values where you will define those secret data which should not be part of any PR but just passed locally from your laptop by `--ocsci-conf path_to_your_ibm_config.yaml`

Reply:
uncommented and checked in

Reply:
reverted back the changes